napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -336,9 +355,10 @@ def _mark_conflicts(self, new_shortcut, row) -> bool:
 
                 # show warning message
                 message = trans._(
-                    'The keybinding <b>{new_shortcut}</b>  is already assigned to <b>{action_description}</b>; change or clear that shortcut before assigning <b>{new_shortcut}</b> to this one.',
+                    'The keybinding <b>{new_shortcut}</b>  is already assigned to <b>{action_description}</b> for the <b>{group}</b> group; change or clear that shortcut before assigning <b>{new_shortcut}</b> to this one.',
```

Comment:
What do you think of:

```suggestion
                    'The keybinding <b>{new_shortcut}</b>  is already assigned to <b>{action_description}</b> in the <b>{group}</b> group; change or clear that shortcut before assigning <b>{new_shortcut}</b> to this one.',
```

Reply:
Also what do we think about removing full stops from viewer action descriptions? E.g.,

https://github.com/napari/napari/blob/a9e7375e339a2240f6cc6affd05eceefa2e843e9/napari/components/_viewer_key_bindings.py#L200

Some actions have a full stop and some don't, so this change would make it consistent. Also the full stop makes the error message above less readable. (Can totally be done in a separate PR, just wanted to suggest it)

Reply:
Yep, I was thinking about that also :+1: Should we create an issue for that then?

Reply:
Good idea, done: #7245